src/dotnet/Chat.UI.Blazor/Components/MentionList/MentionList.razor

```diff
@@ -1,36 +1,50 @@
 @namespace ActualChat.Chat.UI.Blazor.Components
+@using ActualChat.Chat.UI.Blazor.Module
 @inherits ComputedStateComponent<MentionSearchResult[]>
 @{
     var mentions = Mentions;
+    var listCls = "";
     if (mentions.Length < 1)
-        return;
+        listCls += "hidden";
     var selection = Selection;
 }
 
-<div class="@Class mb-1 p-1 border border-mention rounded-md overflow-y-auto max-h-96 w-full">
+<div @ref="Ref" class="@Class @listCls mention-list custom-scrollbar">
     @foreach (var mention in mentions) {
         var searchMatch = mention.SearchMatch;
         var title = searchMatch.Text;
-        var cls = mention == selection ? "bg-04" : "";
-        <div class="@cls px-1 rounded hover:cursor-pointer"
+        var cls = mention == selection ? "bg-mention-list-hover" : "";
+        <div class="@cls hover:cursor-pointer mention-item"
```

Comment:
Can we apply 'cursor-pointer' to 'mention-item'? 'hover' prefix seems not needed.

Reply:
grrrr...... yes, i forgot to do it.